new-lamassu-admin/src/pages/Locales/Locales.js

```diff
@@ -69,6 +69,18 @@ const Locales = ({ name: SCREEN_KEY }) => {
     return saveConfig({ variables: { config } })
   }
 
+  const removeOverridenMachines = data => {
+    if (data) {
+      const overridenMachines = locale.overrides?.map(o => o.machine)
+      const machinesIndex = data.findIndex(o => o.name === 'machine')
+      data[machinesIndex].inputProps.options = data[
+        machinesIndex
+      ].inputProps.options?.filter(m => !overridenMachines.includes(m.deviceId))
+    }
+
+    return data
+  }
+
```

Comment:
This wont work. 

- empty overrides table break the page.
- already existing overrides won't show the machine when being edited.

Take a look at how the FiatBalanceOverrides does it. Better to just follow that logic.